pymc_marketing/mmm/delayed_saturated_mmm.py

```diff
@@ -299,17 +327,17 @@ def build_model(
         --------
         custom_config = {
             'intercept': {'dist': 'Normal', 'kwargs': {'mu': 0, 'sigma': 2}},
-            'beta_channel': {'dist': 'LogNormal', 'kwargs': {'mu': 1, 'sigma': 3}},
-            'alpha': {'dist': 'Beta', 'kwargs': {'alpha': 1, 'beta': 3}},
-            'lam': {'dist': 'Gamma', 'kwargs': {'alpha': 3, 'beta': 1}},
+            'saturation_beta': {'dist': 'Gamma', 'kwargs': {'mu': 1, 'sigma': 3}},
```

Comment:
Why change from LogNormal to Gamma for defaults? Users might see unexpected behavior of old models.

Reply:
Good catch. Is this in docstring only though?

Reply:
`LogNormal` was the value in the docstring but the default distribution was still a `HalfNormal` as in the initial model. Additionally, It was only in the example notebook to show that the config could handle distribution types. 

I'm switching just because `LogNormal` has a greater tail based on the same prior used on the MMM example notebook, and I feel use a Gamma return a more conservative distribution.

Plus, some asked about its use and if, this means that they must make some log transformation. I think it's better to keep it simple, but let me know what do you think?

Reply:
Yes, makes sense! Thanks for the explanation :)